### ISSUE TITLE
Consider adding github issue link to TODOs

### ISSUE THREAD

**Issue by kbeyls** [llsoftsec/llsoftsecbook#156]
Now that we show TODOs also in the HTML output, I think it would be useful to have a link in each todo to a github issue. That probably means we'll also need to create quite a few issues - for TODOs for which currently there are not TODOs.
We could also consider adding a voting button/vote result in a TODOs, so that we can easily collect feedback on which TODOs readers would like to see resolved most. Of course, if we did implement a voting button, it would need to collect the votes on the github issue.

I hope implementing this could help further engage with readers and get more feedback from what readers think.

**Comment by kbeyls** [comment]
Making it easy to add links to issues in TODOs has been implemented in #159 .
The work remaining is to go through the existing TODOs, create issues for the ones that do not have issues yet, and add links to the issues in the book.

I've looked into adding voting mechanisms in the TODOs as shown in the book, but there didn't seem to be any very straightforward way to do so. So, I think I'm going to abandon that idea for now.
